### DISCUSSION TITLE
notebook/natalism

### DISCUSSION THREAD

**Discussion by giscus** [danmackinlay/danmackinlay.name-comments#7] [Comments]
# notebook/natalism

Reproduction and the ethics of population growth and decline

https://danmackinlay.name/notebook/natalism.html

**Comment by veryneant** [comment]
For 2, see also the extremely conservative attitudes to what birth givers are 'allowed' to ingest while pregnant - anything from brie to life-supporting mental health medication.
While on that, see also the concept of medicos 'allowing' and 'not allowing' their patients to follow prescribed behaviours and treatments - ie, the idea that pregnant women and mothers are stupid and incapable of decision making, and on top of that, the concept of Informed Consent being warped in modern hospital settings so that this path is either withheld or acknowledged as a thing that needs to be signed off by a lawyer
HAPPY TO DISCUSS
